src/utils/AMSClientUtils.ts

```diff
@@ -18,6 +18,29 @@ export const whitelistedUrls = [
     "https://ocprodocprodnamgs.blob.core.usgovcloudapi.net"
 ];
 
+export const regionBasedSupportedUrls = [

```

Comment:
Shall we also support dailym?

Reply:
FYI, the list used by CC

https://dev.azure.com/dynamicscrm/OneCRM/_git/CRM.Solutions.ServiceExtended?path=/solutions/ConversationControl/Solution/WebResources/msdyn_chatcontrol.htm&version=GBmaster&line=30&lineEnd=49&lineStartColumn=1&lineEndColumn=48&lineStyle=plain&_a=contents

Reply:
I cross-checked these URLs and they're not used on our end.